internal/drivers/nomad/driver.go

```diff
@@ -468,7 +468,7 @@ func (p *config) destroyJob(vm, nodeID string) (job *api.Job, id string) {
 						Driver:    "raw_exec",
 						Config: map[string]interface{}{
 							"command": "/usr/bin/su",
-							"args":    []string{"-c", fmt.Sprintf("%s stop %s && %s rm %s", ignitePath, vm, ignitePath, vm)},
+							"args":    []string{"-c", fmt.Sprintf("%s stop -f %s && %s rm -f %s", ignitePath, vm, ignitePath, vm)},
```

Comment:
From the documentation, force `rm` will also `kill`(alias for `stop -f`) the VM. So we just need `ignite rm -f vm`

Reply:
> To remove VMs in Ignite, use the following command:

># ignite rm my-vm
>The VM needs to not be running for this to succeed. Using the --force flag (-f for short) a running VM can also be removed, it will be killed before removal. This force option is also able to stop and remove zombie VMs.

> # ignite rm -f my-vm

Reply:
https://github.com/weaveworks/ignite/blob/main/docs/usage.md#removing-a-vm

Reply:
We need stop first for regular cleanup.

Reply:
Where is the regular cleanup happening? I only see force cleanup.